ci/test.sh

```diff
@@ -102,5 +102,13 @@ convox ps stop $ps -a httpd
 convox ps -a httpd | grep -v $ps
 convox deploy -a httpd
 
+# timers
+sleep 30
+
+timerLog=$(convox logs -a httpd --no-follow --since 1m | grep service/web/timer-example)
```

Comment:
Is 30 second sleeping enough?
from schedule `schedule: "* * * * *"` i can see cron should run in at every minutes
also `--since 1m` seems very strict in that sense

Reply:
hey, @nightfury1204. Yes, it has been enough to pass the check.